tests/unit/data/_async/test_mutations_batcher.py

```diff
@@ -1182,3 +1210,25 @@ async def test_customizable_retryable_errors(
                     retry_call_args = retry_fn_mock.call_args_list[0].args
                     # output of if_exception_type should be sent in to retry constructor
                     assert retry_call_args[1] is expected_predicate
+
+    @CrossSync.pytest
+    async def test_large_batch_write(self):
+        """
+        Test that a large batch of mutations can be written
+        """
+        import math
+
+        num_mutations = 10_000
```

Comment:
I think this should test when the mutation size is > batch size limit?

Reply:
It's been a while since I visited this, but it looks like the batch size limit is handled by the flow control, and tested in `test_add_to_flow_max_mutation_limits`. I think this one is just meant to be arbitrarily large

We can make it larger than `_MUTATE_ROWS_REQUEST_MUTATION_LIMIT` if you think that would be good here as well, as an end-to-end type test. But that would probably require mocking that limit constant, because actually processing 100_000 mutations seems to take ~8 seconds, which is fairly long for an individual test